cicd/forgeops-tests/lib/utils/kubectl.py

```diff
@@ -15,6 +15,7 @@
 
 KUBECTL_COMMAND = 'kubectl'
 
+
```

Comment:
PEP 8:
Method definitions inside a class are surrounded by a single blank line.
Extra blank lines may be used (sparingly) to separate groups of related functions